packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -133,7 +144,8 @@ contract L1Block is ISemver, IGasToken {
     ///   7. _blobBaseFee        L1 blob base fee.
     ///   8. _hash               L1 blockhash.
     ///   9. _batcherHash        Versioned hash to authenticate batcher by.
-    function setL1BlockValuesEcotone() external {
+    ///   10. _isDeposit         isDeposit flag
+    function setL1BlockValuesIsthmus() external {
```

Comment:
We should be able to make this function like this:

```solidity
isDeposit = true;
setL1BlockValuesEcotone();
```

Reply:
can you do this with the assembly `calldata` trick?